src/components/Editor/Board/index.tsx

```diff
@@ -32,6 +34,12 @@ export default function Board({ width, height }: { width: number; height: number
     containerRef.current.setTool(tool);
     containerRef.current.drawAll(doc.getRoot().shapes);
 
+    containerRef.current.on('mouseup', () => {
+      if (tool === Tool.Rect) {
+        dispatch(setTool(Tool.Selector));
+      }
+    });
```

Comment:
When destroying the container below, shouldn't the event listeners be removed?

Reply:
> When destroying the container below, shouldn't the event listeners be removed?

I thought that part would be removed by garbage collection, but it would be better to mark it explicitly👍